doc/custom-articles.adoc

```diff
@@ -0,0 +1,31 @@
+= Custom Articles
+:toc:
+
+cljdoc supports Markdown and Asciidoc out of the box but support for additional content types can be added in one of the following ways:
+
+1. Extending the cljdoc codebase itself to cover a new content type
+1. Packaging a separate jar that contains the necessary code to support additional content types
```

Comment:
s/1./2./

Reply:
This kind of numbering is fortunately handled by Markdown and Asciidoc automatically :)